common/admin/nfs/export_test.go

```diff
@@ -74,6 +83,48 @@ func (suite *NFSAdminSuite) setupMockNFSConfig() {
 	require.NoError(err)
 }
 
+func (suite *NFSAdminSuite) setupDirs() {
+	require := suite.Require()
+	conn := radosConnector.GetConn(suite.T())
+	fs, err := cephfs.CreateFromRados(conn)
+	require.NoError(err)
+	defer func() {
+		require.NoError(fs.Release())
+	}()
+	err = fs.Mount()
+	require.NoError(err)
+	defer func() {
+		require.NoError(fs.Unmount())
+	}()
+
+	// establish a "random" list of dir names to work with
+	for _, name := range usableDirNames {
+		err = fs.MakeDir(name, 0777)
```

Comment:
0777 ? Aren't we good with 0755 or 0770 ?

Reply:
Heh.  So I had just been working on an unrelated issue on ceph nfs and I needed to create dirs as 777 because I wanted any and all users to be able to write to those dirs. So 777 kinda stuck in my mind. For this use case I agree that it doesn't matter if we use 777 because we never actually mount or write to these dirs.